spec/system/employee_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe 'Sign in process', type: :system do
+  before do
+    driven_by(:rack_test)
+  end
+
+  it 'enables me to sign in, sign out an employee acount' do
```

Comment:
hey, try to split it into two tests, sign in and log off, I know that you would need to read the documentation of device of how to log in rspec or read this article
https://til.hashrocket.com/posts/2ab67c083b-login-for-feature-test-with-warden-test-helpers

Reply:
I took a look at the link provided and went back to devise, and also divided into sign in, sign out. It's more readable in parts. Thank you